backend/LeetWars.BuilderService/LeetWars.BuilderService.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+</Project>
```

Comment:
your service name should be just Builder
and should contain more files, and implement one of the architectures (Minimal API, The Layered Architectural, CQRS)